app/helpers/users_helper.rb

```diff
@@ -1,2 +1,3 @@
 module UsersHelper
+  
 end
```

Comment:
Yea. They can be very useful, especially if you use the command line flags, but they do make quite the mess.